pkg/apis/garden/v1beta1/helper/helpers.go

```diff
@@ -159,8 +160,8 @@ func GetShootCloudProviderWorkers(cloudProvider gardenv1beta1.CloudProvider, sho
 	return workers
 }
 
-// GetMachineImageFromShoot returns the machine image used in a shoot manifest, however, it requires the cloud provider as input.
-func GetMachineImageFromShoot(cloudProvider gardenv1beta1.CloudProvider, shoot *gardenv1beta1.Shoot) *gardenv1beta1.ShootMachineImage {
+// GetCloudMachineImageFromShoot returns the machine image used in a shoot manifest, however, it requires the cloud provider as input.
+func GetCloudMachineImageFromShoot(cloudProvider gardenv1beta1.CloudProvider, shoot *gardenv1beta1.Shoot) *gardenv1beta1.ShootMachineImage {
```

Comment:
Why is this renamed?

Reply:
Because this functions returns the machine image which is in the Spec -> Cloud -> CloudName -> MachineImage, not the actual machineImages which are used by the Worker(Spec -> Cloud -> CloudName -> []Workers -> []MachineImages)

Reply:
OK, let's call it `GetGlobalMachineImageFromShoot` then. The `worker` section is also below `.spec.cloud.<...>`.

Reply:
Okay